docs/Print Differential.md

```diff
@@ -11,7 +11,7 @@ with brackets to mount to body in [Body Box Mounts](https://cad.onshape.com/docu
 **3D Printing STL Files:**
 * [DiffBrace.stl](../STL/DiffBrace.stl) * 2
 * [DiffEnd.stl](../STL/DiffEnd.stl) * 2
-* [DiffLink.stl](../STL/DiffLink.stl) * 2
+* [DiffLink.stl](../STL/DiffLink.stl) * 2 (Mirror one of them)
```

Comment:
What did you encounter that required mirroring them for your build? The part is symmetric and should have been identical mirrored or not. Curious to learn more details about why mirroring was necessary.

Reply:
Oops. I meant the DiffEnd part. I can push up a correction.

Reply:
Aha, that makes more sense. Yes I agree it's worth calling out DiffEnd is to be mirrored.